features/events/api/src/main/java/org/opennms/netmgt/xml/event/Event.java

```diff
@@ -56,14 +56,15 @@
 
 import org.apache.commons.lang3.builder.ToStringBuilder;
 import org.apache.commons.lang3.builder.ToStringStyle;
+import org.opennms.core.ipc.sink.api.Message;
 import org.opennms.core.network.InetAddressXmlAdapter;
 import org.opennms.core.utils.StringUtils;
 import org.opennms.netmgt.events.api.DateTimeAdapter;
 
 @XmlRootElement(name = "event")
 @XmlAccessorType(XmlAccessType.FIELD)
 //@ValidateUsing("event.xsd")
-public class Event implements Serializable {
+public class Event implements Message,Serializable {
         private static final long serialVersionUID = 6997816158234653400L;
```

Comment:
If you're changing the interfaces it implements, you need to update the `serialVersionUID`.